### ISSUE TITLE
Add "Initialize Interactive with Project" for Portable and DotNet Core Library Projects

### ISSUE THREAD

**Issue by srivatsn** [dotnet/project-system#223]
Ported from https://github.com/dotnet/roslyn/issues/11684 filed by@shaggygi

---

Noticed this feature was available in VS2015, but didn't see as most of my class projects are portable. This would be nice to add to portable and .NET Core v1 projects.

**Comment by hardhub** [comment]
Any chance to get it soon? It is very frustrating when something excellent from previous version is removed in new.

**Comment by reinux** [comment]
> srivatsn added this to the Unknown milestone on Jun 10, 2016

What... How is this not a priority?

**Comment by reinux** [comment]
Are there any workarounds for this?

So far, to use Entity Framework for SQLite, I've loaded all this manually:

```#r "C:\Users\rei\.nuget\packages\microsoft.extensions.caching.abstractions\1.1.1\lib\netstandard1.0\Microsoft.Extensions.Caching.Abstractions.dll"
#r "C:\Users\rei\.nuget\packages\microsoft.extensions.options\1.1.1\lib\netstandard1.0\Microsoft.Extensions.Options.dll"
#r "C:\Users\rei\.nuget\packages\system.interactive.async\3.0.0\lib\netstandard1.0\System.Interactive.Async.dll"
#r "C:\Users\rei\.nuget\packages\remotion.linq\2.1.1\lib\netstandard1.0\Remotion.Linq.dll"
#r "C:\Users\rei\.nuget\packages\microsoft.extensions.caching.memory\1.1.1\lib\netstandard1.3\Microsoft.Extensions.Caching.Memory.dll"
#r "C:\Users\rei\.nuget\packages\microsoft.extensions.logging.abstractions\1.1.1\lib\netstandard1.1\Microsoft.Extensions.Logging.Abstractions.dll"
#r "C:\Users\rei\.nuget\packages\microsoft.extensions.logging\1.1.1\lib\netstandard1.1\Microsoft.Extensions.Logging.dll"
#r "C:\Users\rei\.nuget\packages\microsoft.extensions.dependencyinjection.abstractions\1.1.0\lib\netstandard1.0\Microsoft.Extensions.DependencyInjection.Abstractions.dll"
#r "C:\Users\rei\.nuget\packages\microsoft.extensions.dependencyinjection\1.1.0\lib\netstandard1.1\Microsoft.Extensions.DependencyInjection.dll"
#r "C:\Users\rei\.nuget\packages\system.data.common\4.1.0\lib\netstandard1.2\System.Data.Common.dll"
#r "C:\Users\rei\.nuget\packages\microsoft.entityframeworkcore\1.1.2\lib\netstandard1.3\Microsoft.EntityFrameworkCore.dll"
#r "C:\Users\rei\.nuget\packages\microsoft.entityframeworkcore.sqlite\1.1.2\lib\netstandard1.3\Microsoft.EntityFrameworkCore.Sqlite.dll"
#r "C:\Users\rei\.nuget\packages\microsoft.entityframeworkcore.relational\1.1.2\lib\netstandard1.3\Microsoft.EntityFrameworkCore.Relational.dll"
#r "C:\Users\rei\.nuget\packages\system.diagnostics.diagnosticsource\4.0.0\lib\netstandard1.3\System.Diagnostics.DiagnosticSource.dll"
```

... And it's still asking for more, so I gave up.

**Comment by masaeedu** [comment]
@reinux Maybe a bit off topic, but I personally never use it. It's too stunted; easier to just make a new console application as a scratchpad.

**Comment by shaggygi** [comment]
Not to be a thorn... Just wanted to ping again to add my vote that this is such a great feature.  Hope to get with .NET Core/Standard projects at some point.  CC @davkean

**Comment by tempywritescode** [comment]
@reinux This is why I'm not using it. When this works right it's a crucial tool.

**Comment by davkean** [comment]
Microsofties this PR: https://devdiv.visualstudio.com/DevDiv/_git/CPS/pullrequest/122409?_a=overview makes this feature show up the new project system when the project is targeting .NET Framework. I'm going to file a new bugs against Roslyn to make it work for platforms other than .NET Framework.

**Comment by davkean** [comment]
Filed https://github.com/dotnet/roslyn/issues/26934 for the work to add support for .NET Core/.NET Standard. There is no more work on the project system side, the rest is on Roslyn.

**Comment by davkean** [comment]
This has been merged, dotnet/roslyn#26934 is tracking making this work for .NET Core/.NET Standard.
